README.md

```diff
@@ -1,39 +1,43 @@
 # Mantle: Gluing CoreOS together
 
-This repository is a collection of utilities for the CoreOS SDK.
-
-## plume
-
-CoreOS release utility
-
-## plume index
-
-Generate and upload index.html objects to turn a Google Cloud Storage
-bucket into a publicly browsable file tree. Useful if you want something
-like Apache's directory index for your software download repository.
+This repository is a collection of utilities for developing Container Linux. Most of the
+tools are for uploading, running, and interacting with Container Linux instances running
+locally or in a cloud.
+
+## Overview
+Mantle is composed of many utilies:
+ - `kola` for launching instances and running tests
+ - `ore` for interfacing with cloud providers
+ - `cork` for handling the Container Linux SDK
+ - `gangue` for downloading from google storage
```

Comment:
Google Storage should probably be capitalized here, as it is below.

Also it seems a bit odd that this list is in a different order than the command sections.